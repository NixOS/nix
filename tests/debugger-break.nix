let 
  wat = arg : arg + builtins.break 10;
  foo = arg : arg2 : arg + arg2;
in
foo 12 (wat 11)
