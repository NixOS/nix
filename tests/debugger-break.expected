[34;1m0[0m: while evaluating the file
[34;1m1[0m: while calling [35;1m'foo'[0m
[34;1m2[0m: while calling [35;1m'wat'[0m
[34;1m3[0m: [32;1minfo: [0mbreakpoint reached
